manifests/init.pp

```diff
@@ -61,6 +61,9 @@
 #
 # $qpid_router_logging_path::           Directory for dispatch router logs
 #
+# $docker_registry_port::               Value other than 5000 (default) or 5001 may require SELinux changes
```

Comment:
I still think it would be clearer to prepend the current description with 'Docker registry port.'